frontend/internal-packages/agent/src/chat/workflow/services/workflow.ts

```diff
@@ -55,9 +60,10 @@ const createGraph = () => {
 export const executeWorkflow = async (
   initialState: WorkflowState,
   recursionLimit: number = DEFAULT_RECURSION_LIMIT,
+  log: NodeLogger = () => {},
 ): Promise<WorkflowState> => {
   try {
-    const compiled = createGraph()
+    const compiled = createGraph(log)
```

Comment:
I think adding a logger function to state would allow us to access the logger from state without major code changes, what do you think?

Reply:
Oh, you were right, I moved it to state and removed the unnecessary code!
https://github.com/liam-hq/liam/pull/2041/commits/df7bffc5ff8643c817644a8dd3268144d613d967